styleguide/components/31-molecules/teaser/teaser.twig

```diff
@@ -0,0 +1,107 @@
+<li class="teaser {{ teaser_type }}">
+  <article class="teaser-content">
+    {# title must be the first element in markup #}
+    <div class="content__bottom">
+      <{{ header_level|default('h3') }} id="{{ teaser_uid }}">{{ teaser_title }}</{{ header_level|default('h3') }}>
+    {% if teaser_event_date %}
+      {# Events can consist of two timestamps, so we wrap it in a span. #}
+      <div class="event">
+        <span>
+          <time datetime="{{ teaser_event_date.datetime_1 }}">{{ teaser_event_date.text_1  }}</time>
+          {% if teaser_event_date.datetime_2 and teaser_event_date.text_2 %}
+            &nbsp;<time datetime="{{ teaser_event_date.datetime_2 }}">{{ teaser_event_date.text_2  }}</time>
+          {% endif %}
+        </span>
+      </div>
+    {% else %}
+      <time class="published" datetime="{{ teaser_datetime }}" pubdate="pubdate">{{ teaser_published_date }}</time>
+    {% endif %}
+
+    {% if teaser_topic_links is empty %}
+      {% include '@paragraph' with {
+        'text': teaser_paragraph_text
+      } %}
+    {% endif %}
+
+    {% if teaser_topic_links is not empty %}
+      <ul class="links teaser-topic-links">
+        {% for link in teaser_topic_links|slice(0, max_links|default(5)) %}
+          <li>
+            {% include '@link' with {
+              'text': link.text,
+              'link': link.link
+            } %}
+          </li>
+        {% endfor %}
+      </ul>
+
+
+      {% if teaser_telephone or teaser_email %}
+        <ul class="teaser-contact-info">
+          {% if teaser_telephone %}
+            <li>
+              {% include '@link' with {
+                'text': teaser_telephone,
+                'link': 'tel:'~teaser_telephone,
+                'modifier': 'no-icon'
+              } %}
+            </li>
+          {% endif %}
+          {% if teaser_email %}
+            <li>
+              {% include '@link' with {
+                'text': teaser_email,
+                'link': 'mailto:'~teaser_email,
+                'modifier': 'no-icon'
+              } %}
+            </li>
+          {% endif %}
+        </ul>
+      {% endif %}
+
+      {% if teaser_topic_links|length > max_links|default(5) %}
+        {% include '@link' with {
+          'text': 'show more',
+          'aria_labelledby': teaser_uid,
+          'link': teaser_link_link,
+          'modifier': 'read-more standalone-link'
+        } %}
+      {% endif %}
+    {% else %}
+      {% include '@link' with {
+        'text': 'read more',
+        'aria_labelledby': teaser_uid,
```

Comment:
Can you refactor this to use span.visually-hidden?
It's no longer compliant to WCAG 2.1. 

(there are also 23 duplicate ID violations on the preview page)